README.md

```diff
@@ -1,13 +1,23 @@
-# Daml Platform - Observability Example
+# [Daml Enterprise](https://www.digitalasset.com/products/daml-enterprise) - Observability Example
 
-This repo demonstrates how you can set up your Daml deployment to leverage the observability
-features available, along with a few example Grafana dashboards for monitoring Daml services.
+This project demonstrates a [Daml
+Enterprise](https://www.digitalasset.com/products/daml-enterprise)
+deployment's observability features, along with example Grafana dashboards.  
+
+The project is self contained:  providing scripts to generate requests for
+which metrics are collected for display in the Grafana  dashboards.  
+
+This project is provided for illustrative purposes and local use only. The
```

Comment:
Please keep the disclaimer here that this is not production quality and/or may not fit any use-case (I'd rather avoid the expression 'local use only' as it may be ambiguous local to what):
```suggestion
*Please bear in mind that this project is provided for illustrative purposes only,
and as such may not be production quality and/or may not fit your use-cases.  The
```